n3fit/src/evolven3fit/utils.py

```diff
@@ -0,0 +1,105 @@
+from scipy.interpolate import interp1d
+import numpy as np
+import math
+import pathlib
+import yaml
+import shutil
+
+
+class LhapdfLike:
+    pids_dict = {
+        -6: "TBAR",
+        -5: "BBAR",
+        -4: "CBAR",
+        -3: "SBAR",
+        -2: "UBAR",
+        -1: "DBAR",
+        21: "GLUON",
+        1: "D",
+        2: "U",
+        3: "S",
+        4: "C",
+        5: "B",
+        6: "T",
+        22: "PHT",
+    }
+    pids_order = [
+        "TBAR",
+        "BBAR",
+        "CBAR",
+        "SBAR",
+        "UBAR",
+        "DBAR",
+        "GLUON",
+        "D",
+        "U",
+        "S",
+        "C",
+        "B",
+        "T",
+        "PHT",
+    ]
```

Comment:
Are this dictionaries available from `pdbases` or `lhapdfset`? (so we avoid duplicating content)

Reply:
Not exactly. It is probably more expensive to change the code to use them than just leave the code as it is.